### PR TITLE
add IdlOnlyTypes - even though they are commented out atm

### PR DESCRIPTION
**Pull request by dirk-thomas**
Related to ros2/rosidl#383.

This was mostly used to test `long double` and `wchar` types with FastRTPS. Until the other RMW impl. support this (ros2/rosidl_typesupport_connext#33, ros2/rosidl_typesupport_opensplice#31) the `.idl` file is commented out.

So currently this is a no-op but it allows to easily enable the file for testing in the future.

### CODE REVIEW

**Review by hidmic — COMMENTED** [review]
LGTM though assuming that support in other RMW impl. will be added shortly, would it make sense to not merge this commented out till then?

**Comment by dirk-thomas** [comment]
I doubt that it will be added shortly. Therefore I think it makes sense to merge this as is.

**Comment by hidmic** [comment]
Hmm shouldn't it be added at least to all Tier 1 RMWs?

**Comment by dirk-thomas** [comment]
> Hmm shouldn't it be added at least to all Tier 1 RMWs?

That would be nice. But even then we wouldn't be able to enable it. If at least one RMW impl. doesn't support it the build would fail.

**Comment by hidmic** [comment]
Hmm, I see, we don't have a mechanism to selectively preclude message builds for some RMW typesupports. Alright, fair enough.

**Review by hidmic — APPROVED** [review]
LGTM

**Comment by jacobperron** [comment]
Well, really the file get's built by whoever chooses to use variable `test_interface_files_IDL_FILES` (e.g. test_msgs), so it could alternatively be omitted in the consumer packages, possible based on the current RMW implementation.

**Pull request merged**
